test/e2e/run_test.go

```diff
@@ -304,6 +304,51 @@ var _ = Describe("Podman run", func() {
 
 	})
 
+	It("podman run security-opt unmask on /sys/fs/cgroup", func() {
+
+		SkipIfRootlessCgroupsV1("podman umask on /sys/fs/cgroup will fail with cgroups V1")
```

Comment:
All the currently implemented interactions between unmask and mask are exposed here to be reviewed.